feat: add default support for sphinx-llm (#590)
canonical/sphinx-docs-starter-pack: docs/reference/github-workflows.rst

.. meta::

>     :description: The Starter Pack ships with GitHub workflows to check your documentation's spelling, links, and language in CI.

Reviewer: ```suggestion
    :description: Reference for the built-in GitHub workflows that check your documentation's spelling, links, and language.
```

Author: https://github.com/canonical/sphinx-docs-starter-pack/pull/590/commits/47f855515d8142df3c7341d701d0d9e8c544aecc
